embedding-reports/html5-viewer-telerik-rest-service.md

```diff
@@ -1,14 +1,15 @@
 ---
-title: HTML5 Report Viewer and Reporting REST services
-page_title: HTML5 Report Viewer and Reporting REST services 
-description: HTML5 Report Viewer and Reporting REST services
+title: HTML5 Report Viewer and Telerik REST Service
```

Comment:
Telerik REST Service is too broad, if it means anything at all. Why do we need that change?